packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -111,7 +111,18 @@ class RecoveryPhoneHandler {
       );
       if (result) {
         await this.glean.twoStepAuthPhoneCode.sent(request);
-        return { status: RecoveryPhoneStatus.SUCCESS };
+
+        let nationalFormat: string | null = null;
+        try {
+          nationalFormat = await this.recoveryPhoneService.getNationalFormat(
+            phoneNumber
+          );
+        } catch (e) {
+          // This should not fail since the number was already validated with Twilio but
```

Comment:
I'd remove this... If it shouldn't feel, let's not worry about the try catch.

Reply:
There is just the possibility of a network error here for whatever reason.